libbeat/docs/dashboardsconfig.asciidoc

```diff
@@ -99,3 +99,19 @@ NOTE: This setting only works for Kibana 6.0 and newer.
 
 Force loading of dashboards using the Kibana API without querying Elasticsearch for the version
 The default is `false`.
+
+[float]
+==== `setup.dashboards.retry.enabled`
+
+If this option is set to true, and Kibana is not reachable at the time when dashboards are loaded,
```

Comment:
My comment above made me realise that having a comment about the defaults here would also be great. Some at then of the sentence like. `Default is false.`